src/components/PostComment.jsx

```diff
@@ -0,0 +1,71 @@
+import * as api from "../api.js";
+import React from "react";
+import { useEffect, useState } from "react";
+import { Button, Form, Input, message } from "antd";
+import { useParams } from "react-router-dom";
+
+const { TextArea } = Input;
+
+export default function PostComment({ user, setIsCommentPosted }) {
+  const [postedComment, setPostedComment] = useState("");
+  const [componentDisabled, setComponentDisabled] = useState(false);
+  const [messageApi, contextHolder] = message.useMessage();
+  const { review_id } = useParams();
+  const [form] = Form.useForm();
+
+  const onFinish = () => {
+    api
+      .postComment(review_id, postedComment, user)
+      .then(() => {
+        messageApi.open({
+          type: "success",
+          content: "Comment Posted.",
+        });
+        setPostedComment("");
+        setIsCommentPosted(true);
+        form.resetFields();
+      })
+      .catch((err) => {
+        messageApi.open({
```

Comment:
Really like the use of messageApi here to display the error message in a none intrusive way, it makes for great UI 💯